ix-dev/community/factorio/questions.yaml

```diff
@@ -0,0 +1,359 @@
+                    - value: ""
+                      description: None
+              - variable: port_number
+                label: Port Number
+                schema:
+                  type: int
+                  default: 34197
+                  min: 1
+                  max: 65535
+                  required: true
+              - variable: host_ip
+                label: Host IP
+                description: IP on the host to bind this port. Separate IP adress to prioritize network traffic in the router recommended.
+                schema:
+                  type: string
+                  show_if: [["bind_mode", "=", "published"]]
+                  default: "0.0.0.0"
+                  required: true
+                  $ref:
+                     - definitions/node_bind_ip
+
+        - variable: messaging_port
+          label: Messaging Port
+          schema:
+            type: dict
+            attrs:
+              - variable: bind_mode
+                label: Port Bind Mode
+                description: |
+                  The port bind mode.</br>
+                  - Publish: The port will be published on the host for external access.</br>
+                  - Expose: The port will be exposed for inter-container communication.</br>
+                  - None: The port will not be exposed or published.</br>
+                  Note: If the Dockerfile defines an EXPOSE directive,
+                  the port will still be exposed for inter-container communication regardless of this setting.
+                schema:
+                  type: string
+                  default: "published"
+                  enum:
+                    - value: "published"
+                      description: Publish port on the host for external access
+                    - value: "exposed"
+                      description: Expose port for inter-container communication
+                    - value: ""
+                      description: None
+              - variable: port_number
+                label: Port Number
+                schema:
+                  type: int
+                  default: 27015
+                  min: 1
+                  max: 65535
+                  required: true
+        - variable: host_network
+          label: Host Network
+          description: |
+            Bind to the host network. It's recommended to keep this disabled.
+          schema:
+            type: boolean
+            default: false
+  - variable: storage
+    label: ""
+    group: Storage Configuration
+    schema:
+      type: dict
+      attrs:
+        - variable: volume
+          label: Factorio Server config, mods and saves Storage
+          description: To keep things simple, the container uses a single volume mounted at /factorio. This volume stores configuration, mods, and saves. To change these settings or addings mods or savefiles, access the dataset manually and change the files to your liking.</br>
```

Comment:
I dont think we need to have all this here. You can just have a link to upstream docs.